app/middleware/registerIncomingService.js

```diff
@@ -77,22 +76,11 @@ const validatedService = (serviceId) => {
         registeredServices.map(s => s.serviceId.toLowerCase()).includes(serviceId.toLowerCase()));
 };
 
-const registerIncomingService = (req, res) => {
+const registerIncomingService = (req) => {
     logger.info(req.query);
-
-    return featureToggle.checkToggle('ft_verify_token', req, res)
-        .then(enabled => {
-            if (enabled) {
-                if (verifyToken(req.query)) {
-                    validateParameters(req);
-                }
-            } else {
-                validateParameters(req);
-            }
-        })
-        .catch(err => {
-            req.log.error(err);
```

Comment:
Does the verifyToken still need the catch? What happens if the token is invalid?

Reply:
The catch was for the promise returned by checkToggle. If the token is invalid the parameters do not get validated and then any further request will be redirected to the offline page.